### PR TITLE
ASCellNode selected/highlighted properties

### PR DESCRIPTION
**Pull request by vitalybaev**
Selected/highlighted properties on ASCellNode can be useful for doing some work when table view cell is selected or highlighted (e.g. change node's background color).

### CODE REVIEW

File: AsyncDisplayKit/ASTableView.mm [hunk 1]
```diff
@@ -135,6 +135,15 @@ - (void)didTransitionToState:(UITableViewCellStateMask)state
   [super didTransitionToState:state];
 }
 
+- (void)setSelected:(BOOL)selected
+{
+  _node.selected = selected;
+}
+
+- (void)setHighlighted:(BOOL)highlighted {
```
Comment (appleguy) [inline AsyncDisplayKit/ASTableView.mm]: Bump the { down to the next line for consistency.

**Comment by appleguy** [comment]
@vitalybaev this is a nice extension, thank you!  If you are able to hook it up for ASCollectionView too — which also makes use of ASCellNode — that would be great, and allow me to land the diff.

**Comment by vitalybaev** [comment]
@appleguy I'm sorry, but I'm not familiar with ASCollectionView. Quick look showing me, that we need to subclass UICollectionViewCell, but I don't want break smth.

**Comment by vitalybaev** [comment]
@appleguy check my code please. Now it works with UICollectionView, but I'm not sure if it's right way. Thanks

**Comment by appleguy** [comment]
@vitalybaev looks awesome, thank you!

**Pull request merged**